examples/module-federation/mf-react-component/tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "paths": {
+      "*": ["./@mf-types/*"]
```

Comment:
What it does for?

Reply:
ensure type check in `*.stories.ts`

Reply:
Why does it help?

Reply:
Storybook app like a host app in mf, it will consume rslib module mf assets.  And a remote app will build its own types for host app to ensure type check in host app. If we do not add this in `tsconfig` this import can not pass type check because this is a remote module [code](https://github.com/web-infra-dev/rslib/pull/349/files#diff-d00fa483b327ce5e4ceaac742ff6a53dfa43eeef21cca15d6a7170b848c3d819R2)